api.go

```diff
@@ -106,6 +117,21 @@ func (a *API) NumSlots() (int, error) {
 	return n, unmarshalPyON(s, &n)
 }
 
+// Uptime returns FAH uptime
+func (a *API) Uptime() (time.Duration, error) {
+	s, err := a.ExecEval("uptime")
+	if err != nil {
+		return time.Duration(0), err
```

Comment:
This can be shortened to `return 0, err`.

Reply:
Updated